scripts/remove_operator.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+import subprocess
+import json
```

Comment:
as far as I can see, `json` lib isn't going out of the box with vanilla Python installation, so the `json` dependency installation should be managed somehow, shouldn't it?

Reply:
`json` is a part of the standard library, it's just not imported by default

Reply:
seems, I mix up it with `requests`, sorry for that.